packages/bundler/src/runBundler.ts

```diff
@@ -155,8 +160,14 @@ export async function runBundler (argv: string[], overrideExit = true): Promise<
 
   if (config.rip7560) {
     try {
-      await deployNonceManager(provider, wallet as any)
-      await deployStakeManager(provider, wallet as any)
+      const nonceManager = await deployNonceManager(provider, wallet as any)
+      if (nonceManager.address !== AA_NONCE_MANAGER) {
```

Comment:
Please use a util function or at lest do `.toLowerCase()` consistently when comparing addresses - the amount of time I wasted on these kinds of bugs is just sad.

Reply:
ok for now, but I think its better to make one-pass to return always toChecksum addresses, and then just compare - instead of little the code with huge amount of toLowerCase, which makes it far less readable..